scripts/deploy.sh

```diff
@@ -14,10 +14,23 @@ if [ "$(whoami)" != "zds" ]; then
 fi
 
 if [ "$#" -ne 1 ]; then
-	echo "Usage: $0 <tag name>" >&2
+	echo "Usage: $0 <tag name> <option>" >&2
+	echo "Options:"
+	echo "  --prod (default)"
+	echo "  --beta"
 	exit 1
 fi
 
+# paths
+ENV_PATH="/opt/zds/zdsenv/"
+REPO_PATH="/opt/zds/zds-site/"
+if [[ $1 == "--beta" ]]
+then
+    ENV_PATH="/opt/zdsenv/"
+    REPO_PATH="/opt/zdsenv/ZesteDeSavoir/"
+
+fi
+
 read -p "Did you run specific tasks for this version as described in update.md? [y/N] " -r
```

Comment:
C'était peu le cas puis ça l'a été de plus en plus.

Le 21 juil. 2016 8:48 AM, "victor felder" <notifications@github.com> a
écrit :

> In scripts/deploy.sh
> <https://github.com/zestedesavoir/zds-site/pull/3730#discussion_r71654462>
> :
>
> >  	exit 1
> >  fi
> >
> > +# paths
> > +ENV_PATH="/opt/zds/zdsenv/"
> > +REPO_PATH="/opt/zds/zds-site/"
> > +if [[ $1 == "--beta" ]]
> > +then
> > +    ENV_PATH="/opt/zdsenv/"
> > +    REPO_PATH="/opt/zdsenv/ZesteDeSavoir/"
> > +
> > +fi
> > +
> >  read -p "Did you run specific tasks for this version as described in update.md? [y/N] " -r
>
> J'ai l'impression qu'un bon nombre de ces tâches spécifiques doivent être
> faites pendant la maintenance, pas avant, parce qu'elles nécessitent que le
> code soit à l'état du tag.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zestedesavoir/zds-site/pull/3730/files/f9ec81595ef5480c4accb5c4d9417fee8750edda#r71654462>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFhKnCT1-JRDEzavnNNhak13uagQZNtxks5qXxY_gaJpZM4JRQyy>
> .
>